vuepress/docs/v6.3.2/tutorials/devops/change-default-datasource.md

```diff
@@ -22,11 +22,11 @@ Create a project, ideally in Git repository, and put the following files in the
 It is essential for your custom Docker image to use a file from one of the official Entando Docker base images.
 
 If you intend to use provide your own EAP based image, please use this 
-[standalone-openshift.xml](https://github.com/entando/entando-docker-base-images/blob/master/entando-eap71-clustered-base/standalone-openshift.xml)
+[standalone-openshift.xml](https://github.com/entando/entando-app-engine-base/blob/develop/eap/standalone-openshift.xml)
```

Comment:
@nshaw, please check that this and the following two links to repos are the correct ones.

Reply:
The first 2 look fine.